### ISSUE TITLE
Confusing error message when noxfile.py fails to open a file

### ISSUE THREAD

**Issue by edublancas** [theacodes/nox#307]
Hi, just reporting an error message that could be improved.

**Describe the bug**

When `noxfile.py` attempts to open a file that does not exist, instead of showing the actual traceback, nox displays following error:

`nox > Noxfile /some/path/noxfile.py not found.`

**How to reproduce**

```python
# noxfile.py
with open('file_that_does_not_exist.txt') as f:
    pass
```
Then just run `nox`.

Surprisingly, this does not happen with all exceptions e.g:

```py
# noxfile.py
raise Exception
```
Such `noxfile.py` does not show the "noxfile.py not found" message, but the actual exception, which is what I was expecting.

**Expected behavior**

A message saying there was an error with `noxfile.py`

I'm using version 2019.11.9

**Comment by maciej-lech** [comment]
Hi. This behavior is already fixed in #248

The current message looks like:
```
$ nox                                                                                                                                             
nox > Failed to load Noxfile /home/maciek/.../nox_file_not_found/noxfile.py                                                                                                           
Traceback (most recent call last):                                                                                                                                                                            
  File "/home/maciek/.../nox/nox/tasks.py", line 59, in load_nox_module                                                                                                               
    return importlib.machinery.SourceFileLoader(                                                                                                                                                              
  File "<frozen importlib._bootstrap_external>", line 462, in _check_name_wrapper                                                                                                                             
  File "<frozen importlib._bootstrap_external>", line 962, in load_module                                                                                                                                     
  File "<frozen importlib._bootstrap_external>", line 787, in load_module                                                                                                                                     
  File "<frozen importlib._bootstrap>", line 265, in _load_module_shim                                                                                                                                        
  File "<frozen importlib._bootstrap>", line 702, in _load                                                                                                                                                    
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked                                                                                                                                           
  File "<frozen importlib._bootstrap_external>", line 783, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/home/maciek/.../nox_file_not_found/noxfile.py", line 3, in <module>
    with open('file_that_does_not_exist.txt') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'file_that_does_not_exist.txt'
```

**Comment by V1NAY8** [comment]
Looks like this can be closed now. :)
